README.md

```diff
@@ -1,21 +1,38 @@
+# Macaroon bakery
+
 [![view on npm](https://img.shields.io/npm/v/@canonical/macaroon-bakery?logo=npm)](https://www.npmjs.org/package/@canonical/macaroon-bakery)
 [![npm module downloads](https://img.shields.io/npm/dt/@canonical/macaroon-bakery.svg)](https://www.npmjs.org/package/@canonical/macaroon-bakery)
 
-# bakeryjs
+## Summary
 
-# API Reference
+The macaroon bakery package can be used to create, manipulate and verify [macaroons](http://theory.stanford.edu/~ataly/Papers/macaroons.pdf) without having to worry about how they will be transported or stored. The package provides a set of convenient methods and abstractions that can be easily integrated into any application.
 
-Javascript implementation of the Macaroon Bakery.
+## Installation
+
+```
+npm install @canonical/macaroon-bakery
+```
 
-**Example**
+## Usage
 
 ```js
-import { Bakery } from "@canonical/macaroon-bakery";
+import { Bakery } from "./src/bakery";
```

Comment:
Wouldn't people be importing from `@canonical/macaroon-bakery` when using this library in their projects?